### PR TITLE
[#1006] PROMIS Rollout to Backlog

### PR DESCRIPTION
**Pull request by jhflorey**
this PR address the issue https://github.com/episphere/connect/issues/1006

### CODE REVIEW

File: utils/notifications.js [hunk 1]
```diff
@@ -475,30 +475,6 @@ async function getParticipantsAndSendNotifications({ notificationSpec, cutoffTim
...
-          ...tokenArray,
-          ...emailRecordArray.map((record) => record.token),
-          ...smsRecordArray.map((record) => record.token),
-        ];
-      }
-
-      const tokenSet = new Set(tokenArray);
-      for (const token of tokenSet) {
-        try {
-          await updateSurveyEligibility(token, surveyStatus);
```
Comment (anthonypetersen) [inline utils/notifications.js:494]: can probably remove this function from code since it's not needed anymore

**Review by anthonypetersen — CHANGES_REQUESTED** [review]
see comment

**Review by bransteitterbr — APPROVED** [review]

File: utils/notifications.js (hunk 1 above)
Comment (jhflorey) [reply utils/notifications.js:494]: Solved

**Review by we-ai — APPROVED** [review]
Looks good.

**Pull request merged**